src/services/textChanges.ts

```diff
@@ -1834,10 +1834,11 @@ namespace deleteDeclaration {
 }
 
 // Exported for tests only! (TODO: improve tests to not need this)
+// TODO: knip thinks that this is unused, but it's used in unit tests via `const { deleteNode } = ts.textChanges;`

```

Comment:
Knip can't find the uses of this one, it seems. They come from a destructuring in the tests.

Reply:
Yeah, that's fixed now! Apparently didn't catch a destructured member-accessed imported symbol before. Thanks for the catch.